grpclb/src/test/java/io/grpc/grpclb/GrpclbLoadBalancerTest.java

```diff
@@ -1868,21 +1834,58 @@ public void grpclbBalancerStreamClosedAndRetried() throws Exception {
     inOrder.verify(helper).refreshNameResolution();
 
     // Fast-forward to a moment before the retry, the time spent in the last try is deducted.
-    fakeClock.forwardNanos(10 - 4 - 1);
+    getFastForwardBeforeRetry(10 - 4 - 1, inOrder, expectedInitialRequest);
+
+    // Wrapping up
+    verify(backoffPolicyProvider, times(2)).get();
+    verify(backoffPolicy1, times(2)).nextBackoffNanos();
+    verify(backoffPolicy2, times(1)).nextBackoffNanos();
+    verify(helper, times(4)).refreshNameResolution();
+  }
+
+  private StreamObserver<LoadBalanceResponse> getFastForwardBeforeRetry(int nanos, InOrder inOrder,
```

Comment:
I see the work done by this method aptly suggests it's name to be - fastForwardAndRetry()

Reply:
Renamed the method